### ISSUE TITLE
False positive inspection about arithmetics

### ISSUE THREAD

**Issue by ignatov** [koalaman/shellcheck#1600]
#### For bugs
- Rule Id (if any, e.g. SC2100): 
- My shellcheck version (`shellcheck --version` or "online"): 
The latest master

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/zsh

jdk=my-jdk
echo $jdk

```

#### Here's what shellcheck currently says:

Use $(( )) for arithmetics

#### Here's what I wanted or expected to see:

Nothing?
